internal/jstest/jstest.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jstest
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/FerretDB/dance/internal"
+)
+
+const uri = "mongodb://host.docker.internal:27017"
+
+func Run(ctx context.Context, args []string) *internal.TestResults {
+	// TODO https://github.com/FerretDB/dance/issues/20
+	_ = ctx
+
+	files := []string{}
+	for _, f := range args {
```

Comment:
I think we could treat all arguments as js file names, and add `--verbose` and  `--norc` flags to `dockerArgs`, just like in gotest:

https://github.com/FerretDB/dance/blob/ef96d6eadf48523c482aec0f3cf5331840382ea9/internal/gotest/gotest.go#L32